addons/purchase/migrations/9.0.1.2/post-migration.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# © 2015 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+from openerp import api, SUPERUSER_ID
+
+
+def set_dummy_product(env):
+    products = env['product.product'].search([('name', '=', 'Any product')])
+    if not products:
+        product = env['product.product'].create({
+            'name': 'Any product',
+            'type': 'service',
+            'order_policy': 'manual',
+        })
```

Comment:
idea : maybe set active to false for this dummy product, to avoid to use it after the migration ?